tests/member_chain/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -526,3 +534,121 @@ $window->Data['key']
     });
 
 `;
+
+exports[`staticlookup.php 1`] = `
+<?php
+DB::table('identity')->insert([
+    'ref' => $ref,
+    'handle' => $handle
+]);
+
+DB::table('identity')->insert([
+    'ref' => $ref,
+    'handle' => $handle,
+    'foo' => $bar,
+    'bar' => $foo
+]);
+
+DB::table('identity')->insertReallyReallyReallyReallyReallyReallyReallyReallyReallyLongName([
+    'ref' => $ref,
+    'handle' => $handle,
+    'foo' => $bar,
+    'bar' => $foo
+]);
+
+$page = TableRegistry
+    ::get('Pages')
+    ->findByPath($path)
+    ->find('published')
+    ->find('frontend')
+    ->find('somethingElse')
+    ->first();
+
+Logger
+    ::use_logger('albus')
+    ->info('Albus: failed to find active version for albus object', [
+        'uuid' => $uuid,
+        'requested_date' => $date
+    ]);
+
+DBwithlongname::table('identity')->insertReallyReallyReallyLongName()->someOtherReallyReallyLong();
+
+$DBwithlongname->table('identity')::insertReallyReallyReallyLongName()->someOtherReallyReallyLong();
+
+$page = TableRegistry
+    ::get('Pages')
+    ::findByPath($path)
+    ::find('published')
+    ::find('frontend')
+    ::find('somethingElse')
+    ::first();
+
+$page = TableRegistry
+    ::insertReallyReallyReallyLongName('Pages')
+    ::insertReallyReallyReallyLongName($path)
+    ::insertReallyReallyReallyLongName('published')
+    ::insertReallyReallyReallyLongName('frontend')
+    ::insertReallyReallyReallyLongName('somethingElse')
+    ::insertReallyReallyReallyLongName();
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php
+DB::table('identity')->insert(['ref' => $ref, 'handle' => $handle]);
+
+DB::table('identity')->insert([
+    'ref' => $ref,
+    'handle' => $handle,
+    'foo' => $bar,
+    'bar' => $foo
+]);
+
+DB::table(
+    'identity'
+)->insertReallyReallyReallyReallyReallyReallyReallyReallyReallyLongName([
+    'ref' => $ref,
+    'handle' => $handle,
+    'foo' => $bar,
+    'bar' => $foo
+]);
```

Comment:
Maybe bug in prettier https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBDAdDVAjANnACgB0oACc0gcgEsATBGamAT0pIEp1qoBnOAJxgAlOKly5mIsRKnjJoubJkLl0+WqXMAMtADmAOVQBbQsBIUq-OADNKSUgBIr1gDTmKlABaootfHYdvX3w3Mg9rCAg7R2xUflCLSlj+aIcIiBIAX3YAbhAXEAgAB0ZoHmRQOP4IAHcABTiEcpRUADcIOnyQbH5UMABrOBgAZSK+7l1kGH4AVzgC7j5BOt7dI1RkazE+AoArHgAPACFegaHh4zgtbjhN7fmQMf4l5G6cZlxoLqL+bhgAdToME8yAAHAAGAo-CB8f69IqvH5wJatW4FKwARxm1CsK1Qaw2SC2uB2ID4RmoU1mDx4E3wAEUZhB4HcSQ8sNhAbRgcgAEwFaaoai4CYAYQgRnWryg0DRIBmfAAKjhmsS+JlMkA

need create issue and investigate this

Reply:
Would we expect this instead?

Reply:
@czosel i think yes, but better create issue in prettier before, because they can have cases where it is better output

Reply:
I think it is out of scope this PR, feel free to create issue

Reply:
Agreed I think we should just make output the same as Prettier for now